src/web/nginx.conf

```diff
@@ -90,12 +97,23 @@ http {
     # serve Flask
     location / {
       root /app;
+      proxy_set_header Connection "";
```

Comment:
What's this for?

Reply:
I think that was my attempt to keep the connection alive, but I forgot the reason why I wanted that...